Wikipedia/Code/WMFAuthenticationManager.swift

```diff
@@ -144,6 +144,8 @@ class WMFAuthenticationManager: NSObject {
         }
         SessionSingleton.sharedInstance()?.dataStore.clearMemoryCache()
         
+        SessionSingleton.sharedInstance().dataStore.readingListsController.setSyncEnabled(false, shouldDeleteLocalLists: false, shouldDeleteRemoteLists: false)
```

Comment:
@joewalsh Do these parameters look ok for calling this on logout?

Reply:
yes